libs/remix-ws-templates/src/script-templates/sindri/sindri.conf

```
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://forge.sindri.app/api/v1/sindri-manifest-schema.json",
```

```suggestion
    "$schema": "https://sindri.app/api/v1/sindri-manifest-schema.json",
```
Small tweak. We updated our domains several weeks ago from forge.sindri.app to just sindri.app and we still have some work to do with all of our docs and where this may have been pulled from. It's ok, forge.sindri.app redirects to sindri.app for now.